meson.build

```diff
@@ -675,8 +698,12 @@ subdir('doc/ko')
 subdir('doc/examples')
 subdir('doc/rootfs')
 subdir('hooks')
-subdir('src/lxc/cmd')
-subdir('src/lxc/tools')
+if want_commands
+    subdir('src/lxc/cmd')
+endif
+if want_tools
+    subdir('src/lxc/tools')
+endif
```

Comment:
If we do that, we can then also change src/lxc/tools/meson.build to remove the `if want_tools` section from it.